public/app/features/admin/ServerStatsCard.tsx

```diff
@@ -3,26 +3,56 @@ import React from 'react';
 import Skeleton from 'react-loading-skeleton';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { CardContainer, Stack, useStyles2 } from '@grafana/ui';
+import { Card, Stack, useStyles2, Tooltip, Icon } from '@grafana/ui';
 
 export interface Props {
-  content: Array<Record<string, string | number | undefined>>;
+  content: Array<Record<string, string | number | boolean | undefined>>;
   isLoading?: boolean;
   footer?: JSX.Element | boolean;
 }
 
 export const ServerStatsCard = ({ content, footer, isLoading }: Props) => {
   const styles = useStyles2(getStyles);
   return (
-    <CardContainer className={styles.container} disableHover>
+    <Card className={styles.container}>
       {content.map((item, index) => (
-        <Stack key={index} justifyContent="space-between" alignItems="center">
-          <span>{item.name}</span>
-          {isLoading ? <Skeleton width={60} /> : <span>{item.value}</span>}
-        </Stack>
+        <>
+          {item.tabbed ? (
+            <Stack key={index} justifyContent="space-between" alignItems="center">
+              <span>
+                {' - '}
+                {item.name}
+              </span>
+              {item.tooltip && (
+                <Tooltip content={String(item.tooltip)} placement="left-end">
+                  <Icon name="info-circle" />
+                </Tooltip>
+              )}
```

Comment:
because the parent `Stack` has `space-between`, this ends up floating weirdly in the middle

think you need to wrap the tooltip and item.name element in another `Stack` to keep them together

Reply:
ah, gotcha. got the stack with alignItems to center. Now the tooltip will always be in the middle, regardless of size of screen as well